guide/getting-started/hardware.md

```diff
@@ -68,13 +69,14 @@ You may already be familiar with key fobs from your bank or work. They are small
    height = 400
 %}
 
-Hardware wallets for bitcoin play a similar role but with several differences. Like key fobs, they securely store an authentication mechanism and isolate it from other devices and networks. In bitcoin, these are the seed, associated keys, and all functions that require direct interaction with them.
+Bitcoin hardware wallets play a similar role but with several noteworthy differences. Like key fobs, they securely store an authentication method and isolate it from the Internet, networks, and other devices. In bitcoin, these are the seed, associated keys, and functions that require interaction with them.
```

Comment:
```suggestion
Bitcoin hardware wallets play a similar role but with several noteworthy differences. Like key fobs, they securely store an authentication method and isolate it from the Internet, networks, and other devices. Bitcoin hardware wallets store private keys, seeds, wallet descriptors and other data that allows you to interact with a Bitcoin wallet application externally and construct Bitcoin transactions.
```

Felt like this was a little ambiguous, added some suggestions.

Reply:
Too much complexity, user doesn't know what descriptors are, we barely introed them to BTC, this is getting started page. I mean "wallet descriptors" is way too complicated for this section.

Reply:
We really need to get #59 going and create that page about hardware wallets where we can add these details, and then link to them.